test/e2e/tests/incremental-security.spec.js

```diff
@@ -130,6 +130,7 @@ describe('Incremental Security', function () {
         const revealedSeedPhrase = await driver.findElement(
           '.reveal-seed-phrase__secret-words',
         );
+        await driver.waitForNonEmptyElement(revealedSeedPhrase);
```

Comment:
If we know the seed phrase, can we just use driver.waitForSelector instead, and remove this new method?

Reply:
ty! not in this case, as here we are going to the complete onboarding flow for Create New Wallet, so every time we run it, it's a different seed-phrase

Reply:
Oh my bad, I didn't realise it was creating a new wallet. Thanks for getting back to me